Gemfile

```diff
@@ -12,6 +12,8 @@ gem 'sprockets-rails'
 # Use postgresql as the database for Active Record
 gem 'pg', '~> 1.1'
 
+gem 'devise'
+
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/203624127-e67913de-38d8-4ef3-9bf5-d8b976ceaf12.png)

- [ ] It's great that you have implemented the devise :clap: but the issue is you have missed this thing while implementation that after creating a new user means after sign up it should be confirmed which you can see in the image above :arrow_up: Kindly fix it :blush:

**:arrow_right:  Hint: You can use [letter opener gem](https://github.com/ryanb/letter_opener) for this purpose :blush:**